user/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db               import models
+
+from .validators import (
+    phone_number_validation_function,
+    password_validation_function
+)
+
+class User(models.Model): 
+    name = models.CharField(
+        max_length = 16
+        )
+    birth = models.DateField(
```

Comment:
```suggestion
    birthdate = models.DateField(
```
`birth`라는 변수명은 날짜임을 정확히 인지하기 어려울 것 같아 추천드립니다.